app/actions/index.js

```diff
@@ -33,6 +33,106 @@ export const createDat = () => dispatch => {
   addDat({ path })(dispatch)
 }
 
+export const showDownloadScreen = key => ({
```

Comment:
@juliangruber you mean merge the logic of `showDownloadScreen` into `downloadSparseDat`? I kept it separate because they do two different things? one opens up the screen and second one initiates the download of dat info.